src/UI/Components/ServiceInstanceForm/ServiceInstanceForm.tsx

```diff
@@ -29,9 +29,8 @@ export const ServiceInstanceForm: React.FC<Props> = ({
       : createFormState(fields)
   );
 
-  const getUpdate =
-    (path: string, multi = false) =>
-    (value: unknown): void => {
+  const getUpdate = useCallback(
```

Comment:
Soo... based on what I read on React about this.. is that you technically need to pass the arguments in that array at the end of your hook. 
https://reactjs.org/docs/hooks-reference.html#usecallback

so in our case it would be something like this: 
```

const getUpdate = useCallback(
> () => {
>> //logic
>}, [path, value, multi]
);
```
I'm not so familiar either with this, so I can be wrong and the documentation is kinda abstract.

Reply:
I think that wouldn't work as value is coming from one of different type of children, and we would have to add some state management which is kind of pointless in my opinion. I am curious why I had to use useCallback twice to make error with re-renders disappear

Reply:
Then I fear this is not the right solution somehow 🤔 But then again, I know React can be a pain about rerenders... So, I'm not confident on this matter